edit-post/components/visual-editor/style.scss

```diff
@@ -115,19 +115,6 @@
 	}
 }
 
-// This is a focus style shown for blocks that need an indicator even when in an isEditing state
-// like for example an image block that receives arrowkey focus.
-.edit-post-visual-editor .editor-block-list__block:not(.is-selected) {
```

Comment:
It's a bit hard to explain, there are GIFs in the archives here. But here's a GIF of the behavior today:

![focus](https://user-images.githubusercontent.com/1204802/44390039-ac243c00-a52c-11e8-8aa2-2b1b6c3eec6d.gif)

Back in the day, the "isEditing" mode that fades out UI as you're typing remained faded out even when you navigated into a non text block. This would then get a black outline as I just commented out.

Reply:
Coolio, that makes sense. Good to 🚢 in my mind!

Reply:
Why are we losing the "isEditing" mode when we move to a non-textual block at the moment?

Reply:
I can't recall the details but I believe it was an accessibility thing about the toolbar. I think maybe @aduth worked on it but not sure.

Reply:
Context: #5552

Notably, as otherwise:

" it would be impossible to use the image toolbar by keyboard alone "

Reply:
The function comment for ObserveTyping's `stopTypingOnNonTextField` could probably be improved to make clearer its purpose.